src/components/CollapsibleNavigationMenu.tsx

```diff
@@ -98,29 +93,29 @@ const navItemStyle = css`
   --item-checked-textColor: var(--color-text-0);
 `;
 
-Styled.List = styled(List)`
+const $List = styled(List)`
   gap: 0.5rem;
 `;
 
-Styled.CollapsibleItem = styled(Item)`
+const $CollapsibleItem = styled(Item)`
   ${navItemStyle}
   --item-padding: 0;
 `;
 
-Styled.Sub = styled(Sub)`
+const $Sub = styled(Sub)`
   margin: -0.25rem 0.5rem 0;
   padding-left: 0.5em;
   border-left: solid var(--border-width) var(--color-border);
 
   font-size: 0.92em;
 `;
```

Comment:
I see `as NavItem` for the below `$NavItem`, is the casting not necessary for the other Radix components? `Item`, `Sub`, `List`

Reply:
Nope, since that's the only one with exposed generic type parameters which are the source of the issues.